openwisp_notifications/tasks.py

```diff
@@ -272,7 +272,7 @@ def send_batched_email_notifications(instance_id):
         extra_context = {}
         if notifications_count > display_limit:
             extra_context = {
-                'call_to_action_url': f"https://{current_site.domain}/admin",
+                'call_to_action_url': f"https://{current_site.domain}/admin?open_notification_widget=true",
```

Comment:
Can we use something more user friendly and shorter like `https://{current_site.domain}/admin/#notifications` please?

Reply:
Sure, done.